test/val/val_memory_test.cpp

```diff
@@ -2346,6 +2346,379 @@ OpFunctionEnd)";
   EXPECT_EQ(SPV_SUCCESS, ValidateInstructions());
 }
 
+std::string GenCoopMatKHRLoadStoreShader(const std::string& storeMemoryAccess,
```

Comment:
The tests are failing because this function is unused.

Reply:
Will remove.